mmdet/datasets/pipelines/formating.py

```diff
@@ -75,8 +74,8 @@ def __call__(self, results):
         return results
 
     def __repr__(self):
-        return self.__class__.__name__ + '(keys={}, order={})'.format(
-            self.keys, self.order)
+        return self.__class__.__name__ + \
+            f'(keys={self.keys}, order={self.order})'
```

Comment:
Check if the break line is necessary.

Reply:
Putting them in on line will have max length > 79.